spec/storage/string_interpolation/test.txt

```diff
@@ -0,0 +1 @@
+L#9U"gAc-XLA*/Ac|1M:
```

Comment:
Could you please add a description for this file's purpose so it doesn't get lost?

Reply:
i can't modify the file without messing up the test. I named the folder to give hints on what it's for. Further explanation in the test.